### ISSUE TITLE
Hard to distinguish which input field contain what kind of data

### ISSUE THREAD

**Issue by schiessle** [owncloud/contacts#440]
If you start with a empty contact the placeholder tell you which kind of data should be added to which field. But if you look at a existing contact there is no indication about the input fields data.

For example look at this screenshot:

![contacts](https://cloud.githubusercontent.com/assets/1589737/16306454/0b64471a-395e-11e6-9cb3-b6a3cbaa560b.png)

Can you tell me what data is stored for John Doe?

Here is the answer (but I only know it because I created the input fields and added the data):

From top to the bottom:
- Home phone number (this people might be able to guess, but it could be also any other numeric id, e.g.a fax number)
- work email
- home email
- no, not again my home email address, this time it is my instant message id (jabber/xmpp)
- this is not a second home email address but my federated cloud ID

As you can see this is highly confusing. I think we should have some kind of labels for each input field.

cc @owncloud/designers

**Comment by skjnldsv** [comment]
Do you suggest some header title for the fieldset types?

**Comment by schiessle** [comment]
> Do you suggest some header title for the fieldset types?

Yes, a header title could be a solution. Or a label next to each input field? Maybe "home", "work",... could also be extended to "Home Email", "Work Email",... or something like that? Or a tooltip on hover, but might be also hard to detect and doesn't work on mobile.

I don't have a special solution in mind. I'm happy to let the developers  and designers decide what fits best. But I think there should be definitely something added to distinguish the input fields.

**Comment by jancborchardt** [comment]
Yes, we need to bring back the small header inbetween the fields, like we had in the old Contacts app. :) @skjnldsv do you want to take care of that?

(Should be the size of the text, same opacity/greyscale as the field labels, and left aligned with the fields themselves – not on the far left.)

**Comment by skjnldsv** [comment]
This could be compatible with the way the page is calculated. All the fields are grouped by types.

@jancborchardt I don't have the time right now. But I should be able to work on the app again and continue my work on #411.
#411 is compatible with some heading here. I will see what I can do :)

**Comment by MorrisJobke** [comment]
I like how the android input field labels are shifted up once you type something in:

![screenshot_20160623-163549](https://cloud.githubusercontent.com/assets/245432/16307243/da5cacfe-3960-11e6-8269-555fc3f204ae.png)
![screenshot_20160623-163603](https://cloud.githubusercontent.com/assets/245432/16307244/da5d0f6e-3960-11e6-865e-16f0dc2d85ad.png)

**Comment by skjnldsv** [comment]
I'm on the opposite side :p 
I hate this way of displaying a placeholder. This seems like a new thing a lot of webdev start doing. This confuse the hell out of me! ^^

I think we should keep it simple. :)

**Comment by jancborchardt** [comment]
@MorrisJobke this is not about the same labels though, and also this switching is weird. For addresses e.g. there is a top »Address« label and all the fields like street, city etc have their own labels.

@skjnldsv awesome! Sure, take your time. I would say it should _not_ be part of #411 though since it’s unrelated. And having big pull requests is never ever good! :) Just do a separate pull request for it.

**Comment by skjnldsv** [comment]
Of course :)
